src/allmydata/_auto_deps.py

```diff
@@ -97,6 +97,13 @@
 
     # for 'tahoe invite' and 'tahoe join'
     "magic-wormhole >= 0.10.2",
+
+    # Eliot is contemplating dropping Python 2 support.  Stick to a version we
+    # know works on Python 2.7.  Because we don't have support for `==`
+    # constraints, pin 1.6.x this way.  I feel pretty safe betting that we
+    # won't end up stuck on Eliot 1.6.100 with a critical fix only present in
+    # 1.6.101.  And if we do, I know how to deal with that situation.
+    "eliot >= 1.6.0, <= 1.6.100",
 ]
```

Comment:
i think you might be able to spell that `eliot >= 1.6, < 1.7` ...?

Reply:
Also no support (in Tahoe .... !) for `<` :(

Everything in this list has to go through https://github.com/tahoe-lafs/tahoe-lafs/blob/master/src/allmydata/__init__.py#L305-L333

Possibly we need to do something about this auto_deps stuff?  I think I recall a discussion where this idea was looked upon favorably...